onnxruntime/core/providers/cpu/rnn/deep_cpu_lstm.cc

```diff
@@ -322,8 +376,10 @@ Status DeepCpuLstmOp::ComputeImpl(OpKernelContext& context) const {
   auto& logger = context.Logger();
 
   const Tensor& X = *context.Input<Tensor>(0);  // inputs. [seq_length, batch_size, input_size]
-  const Tensor& W = *context.Input<Tensor>(1);  // weights. [num_directions, 4*hidden_size, input_size]
-  const Tensor& R = *context.Input<Tensor>(2);  // recurrence weights. [num_directions, 4*hidden_size, hidden_size]
+  const Tensor* W = packed_W_.buffer_ ? nullptr : context.Input<Tensor>(1);
+                                                // weights. [num_directions, 4*hidden_size, input_size]
+  const Tensor* R = packed_R_.buffer_ ? nullptr : context.Input<Tensor>(2);
+                                                // recurrence weights. [num_directions, 4*hidden_size, hidden_size]
 
```

Comment:
nit: if the comment needs to be on a separate line should it be before the code it applies to?